classes/web_processor.php

```diff
@@ -32,7 +32,7 @@ class web_processor implements processor {
     protected $profile;
 
     /** @var sample_set */
-    protected $sampleset;
+    public $sampleset;
```

Comment:
Was there any particular reason this is now a **public** scoped variable instead of staying protected?

Reply:
I added a function to sample_set to retrieve the stored maximum, and couldn't access it from the web_processor with protected scope on the sample_set. I'll rework this. It was one part of the attempt to provide as much of a common interface as possible between the completely dissimilar processors.